apps/crn-server/src/routes/team.route.ts

```diff
@@ -25,10 +25,7 @@ export const teamRouteFactory = (
 
     const options = validateFetchOptions(query);
 
-    const result = await teamController.fetch({
-      ...options,
-      showTeamTools: req.loggedInUser?.teams.map((team) => team.id),
```

Comment:
Slightly concerned we could be breaking team tools.

Reply:
It is still working in https://3929.hub.asap.science/ :)

For a team I'm part of -> tools appear
![Screenshot 2023-11-28 at 13 30 08 (2)](https://github.com/yldio/asap-hub/assets/16595804/6099f110-71e2-4821-a4d1-b4ba84b16f77)

For a team I'm not part of -> tools not appearing
![Screenshot 2023-11-28 at 13 30 22 (2)](https://github.com/yldio/asap-hub/assets/16595804/2e7afd89-0a3e-416d-bf16-d8a9735200d4)

Reply:
yeah i think we never really needed tools to be present in the list anyway so now that it was dropped all of this functionality was moved to fetch-by-id endpoint and it just works